components/layout/query.rs

```diff
@@ -640,27 +640,27 @@ pub fn process_resolved_style_request<'a, N, C>(requested_node: N,
     ensure_node_data_initialized(&requested_node);
     ensure_element_styled(requested_node.as_element().unwrap(), style_context);
 
-    let layout_node = requested_node.to_threadsafe();
-    let layout_node = match *pseudo {
-        Some(PseudoElement::Before) => layout_node.get_before_pseudo(),
-        Some(PseudoElement::After) => layout_node.get_after_pseudo(),
+    let layout_el = requested_node.to_threadsafe().as_element().unwrap();
```

Comment:
Turns out to be kinda hard to do because LayoutElement doesn't exist until #13956, and the opaque representations are all Node-y. We can do it later.